source/native/symbulation_sgp.cc

```diff
@@ -36,21 +36,21 @@ int symbulation_main(int argc, char *argv[]) {
   world.CreateDataFiles();
   world.RunExperiment();
 
-  std::pair<emp::Ptr<Organism>, size_t> info = world.GetDominantInfo();
-  std::cout << "Dominant count: " << info.second << std::endl;
+  emp::vector<std::pair<emp::Ptr<Organism>, size_t>> info =
```

Comment:
Since "info" is used quite a bit, let's give it a clearer name, such as "dominant_organisms"?